templates/paragraphs/paragraph--oe-facts-figures.html.twig

```diff
@@ -0,0 +1,16 @@
+{#
+/**
+ * @file
+ * Theme override to display the 'Facts and Figures' #type paragraph.
+ *
+ * @see ./modules/contrib/paragraphs/templates/paragraph.html.twig
+ */
+#}
+{{ pattern('fact_figures', {
+  'style': style,
+  'title': title|default(),
```

Comment:
pls test this default, I've never seen it used like this (not sending a value to the function)